README.md

```diff
@@ -41,113 +41,59 @@ In a new directory where your project will be contained, run:
 npm init react-app <project-name-here>
 ```
 
-#### Install wrld.js
+#### Install wrld-react
 
 ```sh
```

Comment:
Need to make sure the user cd's into <project-name-here> before they install, I think?

Reply:
They don't need to cd into anything here. These are instruction how to set up your own app and install wrld-react